README.md

```diff
@@ -78,6 +78,34 @@ fetching and saving them to local storage can take time (depends on EL RPC host)
 For avoiding `heap out of memory` error, you can pass `NODE_OPTIONS` env var with `--max-old-space-size=8192` value
 and when the application completes its first cycle, you can restart your instance without this env variable.
 
+## Performance optimizations and monitoring the monitor
+
+To tail the logs of your instantiation you can use `docker compose logs -f`. Watch out for connection errors and other issues especially regarding `ethereum-validators-monitoring` process. You can choose to ignore alerting related errors if you haven't configured this feature yet.
+To see a real-time, dynamic view of these containers performance use `docker stats`.
+After the initial launch you can improve the performance, by using some or all of the following options:
+- consider running a local CL+EL.
```

Comment:
Let's add a few words about:
- Increasing CPU on the host on which the application runs
- If running local nodes is not an option for you - reduce possible obstacles between remote nodes and application (firewalls, etc.) - anything that can slow down the connection